modules/ROOT/examples/installation/nginx/subdirectory-configuration.conf

```diff
@@ -138,7 +138,7 @@
               access_log off;
           }
   
-          location ~ /owncloud/.*\.(?:svg|gif|png|html|ttf|woff|ico|jpg|jpeg|map) {
+          location ~ /owncloud/.*\.(?:svg|gif|png|html|ttf|woff|ico|jpg|jpeg|map|json) {
```

Comment:
Quick question; is there any reference to why the prefix `/owncloud/` is used, or is this a defacto convention that I've forgotten?

Reply:
Good question, will take a look